examples/embassy-net-tcp/Cargo.toml

```diff
@@ -15,5 +15,8 @@ embassy-net = { workspace = true, features = ["tcp"] }
 embassy-time = { workspace = true, default-features = false }
 embedded-io-async = "0.6.1"
 heapless = { workspace = true }
-riot-rs = { path = "../../src/riot-rs", features = ["override-network-config"] }
```

Comment:
You only need static IPs in case of usb-ethernet, no? In case of wifi the IP address will be assigned dynamically, so we don't want to override it.

I've added it further below for `cfg(not(context = "esp"))`, but just noticed that the `rpi-pico-w` also uses wifi, so we should also not override it there? Not sure how to do this nicely.

Reply:
This is an issue but out of scope. Currently, the wifi boards will use the same static config by default. We need to fix that. for this pr, keep the "override-network-config".